src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -368,7 +368,7 @@ class GridEditable<
       <GridRow>
         <GridBodyCellStatus>
           <EmptyStateWarning>
-            <p>{t('No results found')}</p>
+            <p>{t('No results found for the given query')}</p>
```

Comment:
Maybe

> No results found for your query

Language feels a bit nicer

Reply:
I feel like "your" implies that the user is responsible for the query, but here the query may not be explicitly provided by the user, like in my initial example of "I just navigated to Performance, what do you mean 'my query'?".

Reply:
Yeah that's true. Something about this wording just feels a little odd to me, but I can't quite place my finger

Reply:
Should a user ever get to a page where there are no results? @EvanPurkhiser works nicely is we assume that the only way they'll see it is if they change the filters/query.

Reply:
Not sure what you're asking here. Performance defaulting to last 24 hours was an example of Sentry automatically inserting a filter for me.

Reply:
@silent1mezzo ^

Reply:
@alxu-sentry Put a different way, landing on performance should never result in `No results found`. That should be true everywhere in the app unless you change it, in which case it's your query that caused no results to be found.